setup.py

```diff
@@ -109,6 +109,9 @@ def get_data_files():
         "easybuild_framework(>=%s.0)" % MAJ_VER,
         "easybuild_easyblocks(>=%s)" % MAJMIN_VER,
     ],
+    extras_require={
+        "archive": ["easybuild-easyconfigs-archive==%s" % VERSION],
```

Comment:
@lexming This will only work if we start publishing `easybuild-easyconfigs-archive` packages on PyPI with those exact versions, I'm not sure that makes much sense?

Reply:
@boegel true, we can just require anything over the current major version for the archive